### ISSUE TITLE
Course list (eg, in cohorts view) should have link elements for better accessibility

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#915]
If there are links, it makes it quicker to navigate by screen reader than the current (button) approach.

Maybe we can make the titles into links, but don't style them any differently than the current plan text.

**Comment by ragesoss** [comment]
This has been fixed.
